### PR TITLE
Add stripe docs

### PR DESCRIPTION
**Pull request by mmiszy**

### CODE REVIEW

**Review by witoszekdev — DISMISSED** [review]

File: docs/payment/stripe.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+# Stripe setup guide
+
+In this guide we'll setup [Stripe](https://stripe.com) payment gateway with Saleor Checkout.
+
+## Setup
+
+1. [Sign up for a Stripe account](https://dashboard.stripe.com/register)
+
+2. Make sure you've enabled at least 1 payment provider in Mollie dashboard (Settings > Payment methods):
```
Comment (peelar) [inline docs/payment/stripe.md:9]:
```suggestion
2. Make sure you've enabled at least 1 payment method in the Stripe dashboard (Settings > Payment methods):
```

I know this is following the same structure as other guides, so I suggest changing the "payment provider" to "payment method" across all of them. Using "payment provider" is confusing here (for me, Stripe is the payment provider).

File: docs/payment/stripe.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
+# Stripe setup guide
+
+In this guide we'll setup [Stripe](https://stripe.com) payment gateway with Saleor Checkout.
```
Comment (peelar) [inline docs/payment/stripe.md:3]:
```suggestion
In this guide, we'll set up [Stripe](https://stripe.com) payment gateway with Saleor Checkout.
```

File: docs/payment/stripe.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+1. [Sign up for a Stripe account](https://dashboard.stripe.com/register)
+
+2. Make sure you've enabled at least 1 payment provider in Mollie dashboard (Settings > Payment methods):
+
+![Getting to settings in Stripe dashboard](../screenshots/setup-stripe-1.png)
+![Getting to payment providers in Stripe settings](../screenshots/setup-stripe-2.png)
+![At least one payment method is enabled on Payment methods page](../screenshots/setup-stripe-3.png)
+
+> Note: Don't worry about "Activation required Please activate your account before accepting live payments with any of the below payment methods." message if you only want to test. Enabled payment methods will still work in test mode, even without completing verification of your Stripe account
```
Comment (peelar) [inline docs/payment/stripe.md:15]:
```suggestion
> Note: Don't worry about the "Activation required Please activate your account before accepting live payments with any of the below payment methods." message if you only want to test. Enabled payment methods will still work in test mode, even without completing verification of your Stripe account.
```

File: docs/payment/stripe.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+3. In your Stripe dashboard, select: Developers -> API keys and copy **Publishable key** and **Secret key**
+
+- **Live API key** - for production environment
+- **Test API key** - for development environment
```
Comment (peelar) [inline docs/payment/stripe.md:20]:
```suggestion
- **Live API key** - for the production environment
- **Test API key** - for the development environment
```

File: docs/payment/stripe.md [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+
+4. Go to Webhooks section in Stripe dashboard (Developers -> Webhooks). You can configure webhooks for your live environment or test them locally.
```
Comment (peelar) [inline docs/payment/stripe.md:22]:
```suggestion
4. Go to the Webhooks section in the Stripe dashboard (Developers -> Webhooks). You can configure webhooks for your live environment or test them locally.
```

File: docs/payment/stripe.md [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+
+- For local development, follow the instructions you see on the screen. `stripe` CLI installation will be required.
```
Comment (peelar) [inline docs/payment/stripe.md:24]:
```suggestion
- For local development, follow the instructions you see on the screen. You will need to install the`stripe` CLI.
```

File: docs/payment/stripe.md [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+- For live environment, you'll need to configure webhooks manually. Click **Add an endpoint** and fill out the form. The endpoint for Stripe webhooks is `<YOUR_CHECKOUT_APP_URL>/api/webhooks/stripe`. Make sure you add the following events:
```
Comment (peelar) [inline docs/payment/stripe.md:25]:
```suggestion
- For the live environment, you'll need to configure webhooks manually. Click **Add an endpoint** and fill out the form. The endpoint for Stripe webhooks is `<YOUR_CHECKOUT_APP_URL>/api/webhooks/stripe`. Make sure you add the following events:
```

File: docs/payment/stripe.md [hunk 8]
```diff
@@ -0,0 +1,49 @@
...
+
+![Webhook configuration](../screenshots/setup-stripe-webhook-1.png)
+
+5. Click "Add endpoint" and then "Reveal secret" to get the webhook secret. You'll need it in the next step.
+
+![Reveal webhook secret](../screenshots/setup-stripe-webhook-2.png)
+
+6. Go to Checkout app configuration and enter the data you've just copied
+
+Configuration page can be found in Saleor dashboard in: **Apps > Third party apps > Checkout**. Once you open the app, select one of the channels and click the gears icon:
```
Comment (peelar) [inline docs/payment/stripe.md:39]:
```suggestion
You can find the configuration page in the Saleor dashboard in: **Apps > Third party apps > Checkout**. Once you open the app, select one of the channels and click the gears icon:
```

File: docs/payment/stripe.md [hunk 9]
```diff
@@ -0,0 +1,49 @@
...
+
+![Payment gateway configuration in Saleor dashboard](../screenshots/config-dashboard-2.png)
+
+To configure the API keys choose "Stripe" from left side navigation and paste what you've copied from Stripe dashboard:
```
Comment (peelar) [inline docs/payment/stripe.md:43]:
```suggestion
To configure the API keys choose "Stripe" from the left side navigation and paste what you've copied from the Stripe dashboard:
```

File: docs/payment/stripe.md [hunk 10]
```diff
@@ -0,0 +1,49 @@
...
+
+![Stripe config inside Saleor dashboard after env variable and profile id were pasted](../screenshots/config-dashboard-stripe.png)
+
+7. 🏁 You've finished the configuration! Now you can enable Stripe for one of your Saleor channels in app configuration:
```
Comment (peelar) [inline docs/payment/stripe.md:47]:
```suggestion
7. 🏁 You've finished the configuration! Now you can enable Stripe for one of your Saleor channels in the app configuration screen:
```

**Pull request merged**